src/tests/system/tests/test_sss_override.py

```diff
@@ -152,49 +156,40 @@ def test_sss_overrides__root_uid_gid_cannot_be_used(client: Client, provider: Ge
     client.sssd.restart()
 
     result = client.tools.getent.passwd("root")
-    assert result is not None
-    assert result.uid == 0
-    assert result.gid == 0
-
-    result = client.tools.getent.passwd("root", service="sss")
-    assert result is None
+    assert result is not None, "root user not found!"
+    assert result.uid == 0, "root uid is not 0"
+    assert result.gid == 0, "root gid is not 0"
 
-    result = client.tools.getent.passwd("user1")
-    assert result is not None
+    assert client.tools.getent.passwd("root", service="sss") is None, "root user is found!"
+    assert not client.tools.getent.passwd("root"), "root user is not found!"
 
 
-@pytest.mark.importance("high")
+@pytest.mark.importance("medium")
 @pytest.mark.topology(KnownTopology.LDAP)
 @pytest.mark.topology(KnownTopology.AD)
 @pytest.mark.topology(KnownTopology.Samba)
 def test_sss_overrides__export_then_import_user_and_group_override_data(client: Client, provider: GenericProvider):
     """
-        :title: Export and import the user and group local override data
+    :title: Export and import the user and group local override data
+    :description: The local override data is stored in the LDB cache
     :setup:
-        1. Create posix user "user1" with posix attributes defined
-        2. Start SSSD
+        1. Create POSIX user "user1" and group "group1"
+        2. Configure SSSD with "ldap_id_mapping = false" and start SSSD
+        3. Create local override for "user1"
+        4. Restart SSSD, this is necessary to enable local overrides
     :steps:
-        1. Override user "user1" to "o-user1"
-        2. Restart SSSD, this is necessary to enable local overrides
-        3. Override group "group1" to "o-group1"
-        4. Export user and group local overrides data to a file
-        5. Delete overrides
-        6. Restart SSSD
-        7. Import user and group local overrides data
-        8. Restart SSSD
-        9. Search for user and group local overrides
+        1. Create override for "group1"
```

Comment:
Why is user over creation part of the setup and group override creation part of the steps?

Reply:
It's mostly to reduce the steps and readability. I don't think it should be included in the steps, if the test is for overrides, let's start the steps doing the override.